cms/static/js/views/pages/container.js

```diff
@@ -191,6 +192,17 @@ define(['jquery', 'underscore', 'gettext', 'js/views/pages/base_page', 'common/j
                 this.duplicateComponent(this.findXBlockElement(event.target));
             },
 
+            moveXBlock: function(event) {
```

Comment:
I think this method should be changed to `moveXBlockModal` as this does not send move request but shows modal.